packages/react-app/src/components/filter.tsx

```diff
@@ -37,10 +37,12 @@ export function Filter() {
           onChange={(val: any) => setFilter({ ...filter, expire: val })}
         />
       </Field>
-      <Field label="Show expired">
-        <Switch
-          checked={filter.showExpired}
-          onChange={(val: any) => setFilter({ ...filter, showExpired: val })}
+      <Field label="Show status">
+        <DropDown
+          id="filterDropdown"
+          items={['Active', 'Expired']}
```

Comment:
Go with ENUM_QUEST_STATE so we can change a status name across the app in a single place